eZ/Publish/Core/Repository/ObjectStateService.php

```diff
@@ -588,28 +585,33 @@ protected function buildDomainObjectStateObject(SPIObjectState $spiObjectState,
                 'names' => $spiObjectState->name,
                 'descriptions' => $spiObjectState->description,
                 'objectStateGroup' => $objectStateGroup,
-            )
+                'prioritizedLanguages' => $prioritizedLanguages,
```

Comment:
Note: `ObjectState` (and `ObjectStateGroup`) uses different naming for some properties. 
There exists `languageCodes` property, however I decided to add `prioritizedLanguages` separately as the `languageCodes` comes from persistence layer and maybe should be left alone. // cc @andrerom 

On the other hand we already have `defaultLanguage` property, so I didn't add extra one `mainLanguageCode`.